spec/requests/video_states_update_spec.rb

```diff
@@ -19,5 +19,14 @@
            params: { input: { key: article.video_code } }
       expect(response).to have_http_status(:unprocessable_entity)
     end
+
+    it "returns not_found if video article is not found" do
+      input = JSON.unparse(input: { key: article.video_code })
```

Comment:
What do you think about using a fake `video_code`? This way we won't have to create and destroy an article and run all the related logic (like callbacks), so the test will be a bit faster.